src/metadatacache/MetadataCache.ts

```diff
@@ -1,23 +1,16 @@
 import { DDO } from '../ddo/DDO'
 import DID from '../ocean/DID'
-import { EditableMetadata } from '../ddo/interfaces/EditableMetadata'
 import { Logger, isDdo } from '../utils'
 import { WebServiceConnector } from '../ocean/utils/WebServiceConnector'
 import { Response } from 'node-fetch'
 import { Metadata, ValidateMetadata } from '../ddo/interfaces'
+const fetchLibrary = require('cross-fetch')
 
 const apiPath = '/api/v1/aquarius/assets/ddo'
 
-export interface QueryResult {
```

Comment:
so, how do we now know outside of ocean.js how the search result is structured? What's the replacement for `QueryResult` in market then?

Reply:
https://github.com/oceanprotocol/market/issues/838 + more discussion on the thread bellow , so we can keep it in one place since it's the same topic.